app/src/main/res/layout/activity_error.xml

```diff
@@ -16,15 +16,15 @@
         android:layout_marginBottom="16dp"
         android:text="TextView"
         android:textColor="#000000"
-        android:textSize="26sp"
+        android:textSize="@dimen/errorHeading"
```

Comment:
We are using 26sp for heading and 18sp for message in both error and login page. So I think we can apply the two text sizes in both page and give them a more generic name in dimens file